File: 2020/1b.jl
#!/usr/bin/env julia

entiers= parse.(Int, readlines(ARGS[1]))
for a in entiers
    for b in entiers
        for c in entiers
            if a+b+c == 2020
                println(a,',',b,',',c,':',a*b*c)
                return a*b*c
            end
        end
    end
end

File: 2021/1b.py
#!/usr/bin/env python3

import sys

with open('input1a.txt') as f:
    aStrProfondeurs = f.read().splitlines()

aProfondeurs = [int(x) for x in aStrProfondeurs]
iMesuresCroissantes = 0
# print(aProfondeurs)
for i in range(len(aProfondeurs)):
    # print(aProfondeurs[i])
    if i >= 3 and aProfondeurs[i] + aProfondeurs[i-1] + aProfondeurs[i-2] > aProfondeurs[i-1] + aProfondeurs[i-2] + aProfondeurs[i-3]:
        iMesuresCroissantes += 1

print(iMesuresCroissantes)
